themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,82 @@
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, you can find answers to some of the most common questions on this page.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<div class="question">
+  <h3>Can I teach Godot in my classroom?</h3>
+  <p>Godot&nbsp;Engine is <a href="https://en.wikipedia.org/wiki/Free_and_open_source_software">free and open source
+      software</a> released under the permissive <a href="/license">MIT license</a>.<br>
+    You can use Godot in your classroom without any restrictions and with zero licensing
+    costs.
+  </p>
+  <p>You can read more about <a href="/license">our license here</a>.</p>
+</div>
+
+<div class="question">
+  <h3>Will the Godot project sign a services agreement provided by my institution?</h3>
+  <p>The Godot Engine project is unable to accept any terms and conditions imposed by organizations, nor will it
+    provide warranties or representations about the Godot Engine. This is clearly outlined in the text of the <a
+      href="/license">MIT license</a>.</p>
+  <p>As a free and open source project, Godot is not backed by a corporation or any individual. Accordingly, it doesn't
+    have the legal capacity to enter into agreements, even if such agreements did not conflict with the MIT license.</p>
+</div>
+
+<div class="question">
+  <h3>What data does the Godot Engine executable collect from users?</h3>
+  <p>When you use the Asset Library in the Godot Engine application, HTTPS requests are made to the Asset Library
+    website, and will contain: Godot Engine version, search query, selected assets, and the user's IP address. The Asset
+    Library
+    processes this data without storing it, but our TuxFamily hosting's HTTP server may log these requests. Such logs
+    are subject to TuxFamily's privacy policy (https://faq.tuxfamily.org/Privacy/En).<br>
+    This is the only personal data collected by the Godot Engine application.</p>
+  <p>To be clear, the Godot Engine executable does not ask for or otherwise collect personally identifying information
+    from
+    its users. The HTTPS requests include the bare minimum amount of information required to do an online search and
+    transmit substantially less information than most internet browsing will.</p>
+</div>
+
+<div class="question">
+  <h3>I don't have permission to install software on my devices. How can I use Godot?</h3>
+  <p>You can use the Godot Engine on the browser without the need to install it on your device.<br>This is useful if you
+    are restricted by your organization or limited by the device being used and you can't install software locally</p>
+  <p>Try the <a href="https://editor.godotengine.org/releases/latest/">web editor now</a>.</p>
```

Comment:
Note that there may be more implications to a educational institution using Godot in the browser; students may upload their projects/data to a website now, or at least entrust it to a browser. Not sure if its worth mentioning this + that you can host yourself locally.

Reply:
Yes, let's leave this as is. My understanding is that using the Godot Engine in browser carries the same privacy implications as using the executable i.e. we don't do anything server-side other than provide the application which runs on the users hardware. They certainly aren't uploading their data to our servers